live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/transform/request/ATSTransformRequester.java

```diff
@@ -96,9 +104,9 @@ private Map<String, String> getTransformRequestOptions(Transform transformProper
         return Map.of(TIMEOUT_KEY, String.valueOf(transformProperties.request().timeout()));
     }
 
-    private String makeClientDataString(TransformRequest transformRequest)
+    private String makeClientDataString(TransformRequest transformRequest, int attempt)
```

Comment:
At first glance I've red "mockClientDataString" and was confused, and started even wondering if is it a test 😂

Reply:
I think I we get rid of this method... I'll check ;)